src/Symfony/Component/Security/Core/Authorization/TraceableAccessDecisionManager.php

```diff
@@ -47,10 +47,16 @@ public function decide(TokenInterface $token, array $attributes, $object = null)
     {
         $result = $this->manager->decide($token, $attributes, $object);
 
+        $strategy = 'unknown';
+        if ($this->manager instanceof AccessDecisionManager) {
+            $strategy = $this->manager->getStrategy($token, $attributes, $object);
```

Comment:
I don't like the fact that there is a double call to the `getStrategy` method (the other in the `decide` call), but I don't see how else I can get the strategy (unless the last strategy used get stored in the `AccessDecisionManager`)